app/Domain/Clients/Templates/delClient.blade.php

```diff
@@ -34,8 +34,7 @@
                 <x-global::forms.button type="submit" name="del" class="button">
                     {{ __('buttons.yes_delete') }}
                 </x-global::forms.button>
-
-                <x-global::forms.button tag="a" href="/clients/showClient/{{ $client['id'] }}" class="btn btn-primary">
+                <x-global::forms.button tag="a" href="/clients/showClient/{{ $client->id }}" class="btn btn-primary">
```

Comment:
We should not need the class definitions here since they are already part of the component.

Reply:
While we're at it, can you change the "class="button" from above as well. 
Then lastly the secondary button should have content-role="tertiary"